src/AdminService.Data.Provider.MsSql.Ef/Migrations/20211123223000_InitialTable.cs

```diff
@@ -0,0 +1,257 @@
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "ImageService", true, null, null });
+
+      builder.InsertData(
+        table: DbServiceConfiguration.TableName,
+        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "HistoryService", true, null, null });
+
+      builder.InsertData(
+        table: DbServiceConfiguration.TableName,
+        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "FileService", true, null, null });
+
+      builder.InsertData(
+        table: DbServiceConfiguration.TableName,
+        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "EducationService", true, null, null });
+
+      builder.InsertData(
+        table: DbServiceConfiguration.TableName,
+        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "DepartmentService", true, null, null });
+
+      builder.InsertData(
+        table: DbServiceConfiguration.TableName,
+        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "CompanyService", true, null, null });
+
+      builder.InsertData(
+        table: DbServiceConfiguration.TableName,
+        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
+        columnTypes: new string[]
+        {
+          "uniqueidentifier",
+          "nvarchar(max)",
+          "bit",
+          "uniqueidentifier",
+          "datetime2"
+        },
+        values: new object[] { Guid.NewGuid(), "AchievementService", true, null, null });
```

Comment:
you can use [this InsertData() method](https://docs.microsoft.com/ru-ru/dotnet/api/microsoft.entityframeworkcore.migrations.migrationbuilder.insertdata?view=efcore-5.0#Microsoft_EntityFrameworkCore_Migrations_MigrationBuilder_InsertData_System_String_System_String___System_String___System_Object____System_String_) for adding several lines in one method
it will look something like this:
builder.InsertData(
        table: DbServiceConfiguration.TableName,
        columns: new[] { "Id", "ServiceName", "IsActive", "ModifiedBy", "ModifiedAtUtc" },
        columnTypes: new string[]
        {
          "uniqueidentifier",
          "nvarchar(max)",
          "bit",
          "uniqueidentifier",
          "datetime2"
        },
        values: new object[,] 
       {
         { Guid.NewGuid(), "AchievementService", true, null, null },
         {Guid.NewGuid(), "UserService", true, null, null}
       });

Reply:
fixed